terminal.go

```diff
@@ -1457,6 +1457,11 @@ func (g *LightningTerminal) shutdownSubServers() error {
 		}
 	}
 
+	if err := g.sessionDB.Close(); err != nil {
```

Comment:
Should also check that `g.sessionDB != nil`, incase `g.start` errors before setting the `sessionDB` field.

Reply:
def - good catch 🙏